### DISCUSSION TITLE
UUID instead of big integers in ids

### DISCUSSION THREAD

**Discussion by antonkomarev** [cybercog/laravel-love#39] [Ideas]
Need to think if it's possible to use UUID in all model's ids without losing performance a lot.

Some databases like PostgreSQL supports UUID data types, so we need to have ability to choose if UUID generation should be done at the application or database level.

**Comment by jayenne** [comment]
I use UUIDs for the 'user' model (and considering the 'posts' model too). is There a migration set available to either change the tables post migration or a new set of migrations available for UUID's or CHAR(36)?

should I be modifying my own from the vendor defaults?

**Comment by antonkomarev** [comment]
@jayenne this discussion is about UUIDs for ids inside of the package: Reaction, ReactionType, Reacter, Reacter and so on.

If you have User model with identifier of UUID type - this package will work out of the box and doesn't require to make any changes because it doesn't store user id inside of the package database tables.

If you will have any problems with integration — create an issue.
